src/mlpack/tests/image_load_test.cpp

```diff
@@ -67,6 +70,110 @@ BOOST_AUTO_TEST_CASE(SaveImageAPITest)
     BOOST_REQUIRE_EQUAL(im1[i], im2[i]);
 }
 
+/**
+ * Test if the image is saved correctly using API for transpose.
+ */
+BOOST_AUTO_TEST_CASE(SaveImageTransposeAPITest)
+{
+  data::ImageInfo* info = new ImageInfo(5, 5, 3, 90);
+
+  arma::Mat<unsigned char> im1;
+  size_t dimension = info->Width() * info->Height() * info->Channels();
+  im1 = arma::randi<arma::Mat<unsigned char>>(dimension, 1);
+  BOOST_REQUIRE(data::Save("APITest.bmp", im1, *info, false, false) == true);
+
+  arma::Mat<unsigned char> im2;
+  BOOST_REQUIRE(data::Load("APITest.bmp", im2, *info, false, false) == true);
+
+  BOOST_REQUIRE_EQUAL(im1.n_cols, im2.n_cols);
+  BOOST_REQUIRE_EQUAL(im1.n_rows, im2.n_rows);
+  for (size_t i = 10; i < im1.n_elem; ++i)
```

Comment:
Not sure, why do we start from index 10?

Reply:
All other test were written in the same way and hence I thought to be consistent. I made changes in each of them.

Reply:
Okay, thanks!